app/views/bulk_update/trainees/review_errors/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= render PageTitle::View.new(text: "Upload summary") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: "Home", href: root_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= register_form_with(model: @bulk_add_trainee_upload_form, url: bulk_update_trainees_uploads_path) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-xl"><%= organisation_name %></span>
+        Review errors for <%= @bulk_update_trainee_upload.trainee_upload_rows.joins(:row_errors).count("DISTINCT(bulk_update_trainee_upload_rows.id)") %> trainees in the CSV that you uploaded
```

Comment:
1. There is an association on the `BulkdUpdate::TraineeUpload#row_errors`.
2. Do we need `DISTINCT`?
3. I suggest to use `size` instead of `count` as the latter will do execute a query instead of using the in memory collection.